src/components/PokeCard.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function PokeCard ({ pokemon }) {
```

Comment:
Good but it would be even better if you pass the properties of the pokemon obj directly to the component
```suggestion
export default function PokeCard ({ name, sprites, video }) {
```
In this case, you would get data to use lines below in a way that is easier to read.
Also you would use the component like in:
```suggestion
<PokeCard name={name} sprites={sprites} video={video} />
```
There are not so many elements to pass to the component yet to encapsulate into an obj.